fedopt_guide/cifar10/centralized_cifar10.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020, Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Baseline experiment on centralized CIFAR-10 data."""
+
+from typing import Any, Mapping, Optional
+
+import tensorflow as tf
+
+from utils import centralized_training_loop
+from utils.datasets import cifar10_dataset
+from utils.models import resnet_models
+
+
+CIFAR_SHAPE = (32, 32, 3)
+NUM_CHANNELS = 3
+NUM_CLASSES = 100
+
+
+def run_centralized(optimizer: tf.keras.optimizers.Optimizer,
```

Comment:
Can't we simply use tensorflow datasets to load CIFAR-10 here? It seems wasteful to load it, partition, then re-amalgamate.

Reply:
Yes that makes sense, it would definitely become a lot simpler!